source/switching_to_live/set_up_a_live_epdq_account/index.html.md.erb

```diff
@@ -1,218 +1,145 @@
 ---
-title: Switching to live
-weight: 100
+title: Connect your live account to ePDQ
+weight: 300
 ---
 
-# Set up a live ePDQ account
+# Connect your live account to ePDQ
 
-Please read the guidance on what to do [before you switch to
-live](/switching_to_live/before_you_switch_to_live/#before-you-switch-to-live)
-first.
+Before you start, you must:
 
-You should follow these instructions in order.
+- read about [what to do first](/switching_to_live/#before-you-start)
+- [request a live GOV.UK Pay account](/switching_to_live/#1-request-a-live-gov-uk-pay-account)
 
-You can also read about setting up live
-[Smartpay](/switching_to_live/set_up_a_live_smartpay_account) and
-[Worldpay](/switching_to_live/set_up_a_live_worldpay_account) accounts.
+You must also be an admin on both your:
 
-## Add payment methods to your account
-
-Sign in to the [ePDQ admin
-site](https://payments.epdq.co.uk/Ncol/Prod/BackOffice/login/index). On the
-landing page, go to __Configuration__ then __Payment methods__ and select __Choose
-new payment methods__. Select __Add__ next to all relevant payment methods.
+- GOV.UK Pay account
+- ePDQ account
 
-On the __Contract data__ tab:
+## Add payment methods to your account
 
-1. Specify whether you have signed a contract for distance selling with an acquiring bank
+1. Sign in to your [ePDQ account](https://payments.epdq.co.uk/Ncol/Prod/BackOffice/login/index).
 
-2. Complete the __Affiliation number (UID/Merch ID/VP number)__ field
+2. Select the __Configuration__ tab, then __Payment methods__. Check that Visa and Mastercard have already been added by default.
 
-3. Select __Submit__
+3. Select __Choose new payment methods__.
 
-On the __PM Activation__ tab:
+4. Select __Add__ next to the card types you want to accept.
 
-1. Select __Yes__ for __Activation__
+5. Complete the fields on the __Contract data__ tab.
 
-2. Select __Submit__
+6. When you reach the __PM Activation__ tab, select __Yes__ for __Activation__. Then select __Submit__.
```

Comment:
Sorry, I don’t think I was clear enough. They have to fill in the _Contract data_ form and click the _Submit_ button (literally “SUBMIT” on ePDQ’s site) on that tab, which causes the page to load again with the _Contract data_ tab still selected. Only _then_ can they switch to the _PM activation_ tab!